File: java-concert/src/main/java/prac/javaconcert/concert/request/ConcertSaveRequest.java
package prac.javaconcert.concert.request;

import lombok.Getter;
import lombok.NoArgsConstructor;
import prac.javaconcert.concert.domain.Genre;
import prac.javaconcert.concert.domain.Place;
@Getter
@NoArgsConstructor
public class ConcertSaveRequest {
    private String name;


    // 콘서트 포스터
    private String poster;

    // 콘서트 장소
    private String region;

    private String location;

    // 콘서트 기간
    private String period;

    // 콘서트 시간
    private Integer runningTime;

    // 콘서트 관람 연령
    private String ageLimit;

    // 콘서트 장르
    private Genre genre;

    // 공연 정보
    private String Description;
}

File: concert/src/main/kotlin/prac/concert/request/ConcertSaveRequest.kt
package prac.concert.request

import prac.concert.domain.Genre
import prac.concert.domain.Region

data class ConcertSaveRequest(
    val name: String = "",
    val poster: String = "",
    val region: Region = Region.SEOUL,
    val location: String = "",
    val period: String = "",
    val runningTime: Int = 0,
    val ageLimit: String = "",
    val genre: Genre = Genre.BALLADE,
    val description: String = ""
)
